client/src/components/header/CartIcon/CartIcon.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import Cart from '@material-ui/icons/ShoppingCart';
+import styles from './CartIcon.module.scss';
+import { useSelector } from 'react-redux';
+import CartPopup from '../../cart/CartPopup/CartPopup';
+
+function CartIcon() {
+  const parts = useSelector((state) => state.cart.items);
+  const amount = parts.reduce((acc, cur) => {
+    return acc + cur.amount;
+  }, 0);
+  const [show, setShow] = useState(false);
```

Comment:
Boolean flags are commonly named with `is` prefix. For example, `const [isVisible, setIsVisible] = ...`